frank/radial_fitters.py

```diff
@@ -20,16 +20,14 @@
    of deprojected visibities.
 """
 
+from frank.hankel import DiscreteHankelTransform
```

Comment:
here another case of importing frank before standard packages. risky

Reply:
cool, we'll do this in another PR!